Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/HDInsightNewAPI/ClusterOperationNewAPIImpl.java

```diff
@@ -69,11 +69,22 @@ private Observable<ClusterConfiguration> getClusterConfigurationRequest(
                 .withUuidUserAgent()
                 .post(url, entity, null, null, ClusterConfiguration.class);
     }
+    private Observable<ClusterConfiguration> getClusterConfigurationRequestFormNewAPI(
```

Comment:
Could we just replace the implementation of `getClusterConfigurationRequest`, or we need to keep both the two methods for different cases? 
If we need to keep both of the methods, shall we rename the method to `getClusterConfigurationFromManagementEndpoint` and `getClusterConfigurationFromResourceManagerEndpoint`? Or we could extract a method and pass endpoint as a parameter

Reply:
got it